### PR TITLE
Add backend support for ConstantOfShape

### PR DESCRIPTION
**Pull request by chinhuang007**
Add backend support for ConstantOfShape which replaced ConstantLike
in ONNX

### CODE REVIEW

**Comment by chinhuang007** [comment]
@tjingrant ConstantLike has not made to any formal ONNX releases. I don't see backward compatibility issues. All our backend handlers take care of one of the ONNX ops, specified in @onnx_op("xxxx"). Since ConstantLike is not in an ONNX release, I don't think it makes sense to leave it in onnx-tf. Any ONNX models containing ConstantLike should change to using ConstantOfShape or it will not pass the checker and will cause confusion.

**Comment by tjingrant** [comment]
@chinhuang007 thanks, makes sense.

File: doc/support_status.md [hunk 1]
```diff
@@ -174,9 +175,11 @@ ______
 |Floor|1, 6|
 |FloorDiv|1|
 |FusedBatchNorm|1, 6, 7|
+|GRU|1, 3, 7|
```
Comment (tjingrant) [inline doc/support_status.md]: @fumihwh what can we do to make it obvious that these support are experimental?

**Comment by tjingrant** [comment]
Hi @chinhuang007 , could you please rebase this PR against master once this PR https://github.com/onnx/onnx-tensorflow/pull/359 is merged?

The reason for the delay is because I want to make it rather explicit that RNN frontend support is experimental.

**Comment by chinhuang007** [comment]
@tjingrant Sure, I will rebase once PR #359 is merged.

**Comment by chinhuang007** [comment]
@tjingrant Rebased, please review.

**Pull request merged**